product_code/build.sbt

```diff
@@ -64,6 +64,7 @@ lazy val hyperledger_component = (project in file("hyperledger_component"))
 
 lazy val course_service_api = (project in file("course_service/api"))
   .settings(Settings.apiSettings("course_service_api"))
+  .settings(libraryDependencies += Dependencies.uuid)
```

Comment:
We only need this in the Tests, right?
Are we able to add the `% Test` suffix here?